### PR TITLE
Rename variables

### PR DESCRIPTION
**Pull request by DawidPiechota**

### CODE REVIEW

File: client/src/components/ForceGraph.vue [hunk 1]
```diff
@@ -468,16 +468,17 @@ export default {
...
       // so we don't scroll indefinitely
-      const graphBox = this.selections.graph.node().getBBox();
-      const margin = 200;
-      const worldTopLeft = [graphBox.x - margin, graphBox.y - margin];
-      const worldBottomRight = [
-        graphBox.x + graphBox.width + margin,
-        graphBox.y + graphBox.height + margin
-      ];
-      this.zoom.translateExtent([worldTopLeft, worldBottomRight]);
+      // const graphBox = this.selections.graph.node().getBBox();
```
Comment (baatochan) [inline client/src/components/ForceGraph.vue:474]: Why are you commenting it out? If that doesnt work consider removing that completly from master

File: client/src/components/VisualiseData.vue [hunk 2]
```diff
@@ -99,15 +99,15 @@ export default {
     },
     mergeLinks(indexA, indexB) {
       for (let i = 0; i < this.changedData.links.length; i++) {
-        let dupa = 0;
```
Comment (baatochan) [inline client/src/components/VisualiseData.vue:102]: why was it even merged with this dupa XD

Comment (DawidPiechota) [reply client/src/components/VisualiseData.vue:102]: I had no good idea on how to name this variable, so I thought ill change it later. And then forgot about it completely xd

File: client/src/components/ForceGraph.vue (hunk 1 above)
Comment (DawidPiechota) [reply client/src/components/ForceGraph.vue:474]: I was going to try fixing it, but removing it for good seems like a better option

File: client/src/components/VisualiseData.vue (hunk 2 above)
Comment (baatochan) [reply client/src/components/VisualiseData.vue:102]: Well I understand you, Im just curious why anyone in review havent noticed that

**Pull request merged**